Charter.md

```diff
@@ -148,45 +149,57 @@ Meetings may be restricted to Community Group participants, but a public
-decision reached at an online or in-person meeting within 7 days of
-publication of the decision provided that they include clear technical
-reasons for their objection. The Chairs will facilitate discussion to
-try to resolve the objection according to this decision process.
+## Decision Process
+
+This group will seek to make decisions where there is consensus, as described in this section.
+
+### Work item owner
+
+* Every work item (crate or report) worked on by the Community Group will have a list of owners which will always include the Chairs.
+* This list of owners will be recorded using GitHub's [CODEOWNERS] mechanism.
+* Only the Chairs are the owners of the CODEOWNERS file.
+* Any Community Group participant can ask to be promoted owner of a work item once a pull-request they have submitted on this work item is merged
+  (this rule is loosely inspired by the [pull-request hack]).
+* The Chairs may demote an owner of a work item who has not contributed (with pull-requests or reviews) to that item in the past year.
+
+### New work item
+
+* Any Community Group participant may propose a new work item (crate or report) by making a pull-request on the Community Group's GitHub repository
+  with the label `new-work-item`, which constitutes a call for consensus.
+* No less than two weeks after the pull-request was posted,
+  the Chairs assess consensus based on the feedback on the pull-request,
+  and record a group decision on GitHub, following [Section 5.2] of the W3C process.
+* If the decision is to accept the new work item, the Chairs merge the pull-request and promote its author owner of the new work item.
+* Regardless of the consensus of its participant, the Community Group can not accept a work item that is not in its [scope](#scope-of-work),
+  as defined by this charter.
+
+### Merging pull-requests
+
+* Every Community Group participant will have [write access] to the group's GitHub repository,
+  but merging a pull-request will require approval by at least two owners of the affected work item.
+* To ensure the applicability of the rule above,
+  pull-requests spanning multiple work items are strongly discouraged,
+  but the Chairs may decide to allow them on a per-case basis.
+* Once sufficiently approved by the appropriate owners (as defined above),
+  a pull-request can be merged even if there is disagreement about it.
+  Individuals who disagree with the choice are strongly encouraged to take ownership of their objection by submitting a subsequent pull-request.
+
+### Publishing work
+
+* The Chairs are collectively the owners on [crates.io] of the crates released by the Community Group.
+* To request the release of a crate on crates.io, or the publication of a report as a [Final Community Group Report],
+  a Community Group participant opens a pull-requests or an issue with the label `request-publication`,
+  which constitutes a call for consensus.
+* No less than two weeks after the pull-request / issue was posted,
```

Comment:
Two weeks sounds very long in case the release is just publishing a bug fix.

Moreover, it might even be harmful if the CG owns some utility library crates that might have some security vulnerability (we will be in this case if the CG host an IRI validation library).

What about stating this rule only if the release contains "substantial changes" (opposite to just a bug fix), so some definition of "substantial".

Reply:
Good point. I will suggest below some exceptions to this rule.